Backend/azure_open_ai_service.py

```diff
@@ -16,7 +16,13 @@ class AzureOpenAIService:
     Azure OpenAI Service wrapper for chat completion.
     """
 
-    def __init__(self, ai_endpoint: str, assistant_id: str):
+    def __init__(self):
+        """Initialize service without specific configuration."""
+        self.project_client = None
+        self.assistant_id = None
+
+    def configure(self, ai_endpoint: str, assistant_id: str):
+        """Configure the service with endpoint and assistant ID."""
```

Comment:
This isn't needed. ai_endpoint is always the same once startup happens. assistant_id can be passed in on the get_response.